Sources/Document/Json/JSONTree.swift

```diff
@@ -140,6 +142,36 @@ func createCRDTTreeNode(context: ChangeContext, content: any JSONTreeNode) throw
     return root
 }
 
+/**
+ * `validateTextNode` ensures that a text node has a non-empty string value.
+ */
+func validateTextNode(_ textNode: JSONTreeTextNode) throws {
+    if textNode.value.isEmpty {
+        throw YorkieError.unexpected(message: "text node cannot have empty value")
+    }
+}
+
+/**
+ * `validateTreeNodes` ensures that treeNodes consists of only one type.
+ */
+func validateTreeNodes(_ treeNodes: [any JSONTreeNode]) throws {
```

Comment:
If the following works,
https://github.com/yorkie-team/yorkie-ios-sdk/pull/93#discussion_r1306850914
then consider changing from `any -> some`

Reply:
If you apply this, it will not be compiled.